src/app/dashboard/log-report/log-report.component.ts

```
@@ -1,79 +1,233 @@
+        start = today.subtract(6, 'days').startOf('day');
+        end = today.endOf('day');
+        break;
+      case 'lastMonth':
+        start = today.subtract(1, 'month').startOf('day');
+        end = today.subtract(1, 'month').endOf('day');
+        break;
+      case 'last30Days':
+        start = today.subtract(30, 'days');
+        end = today.endOf('day');
+        break;
+      case 'custom':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)